data/moves.js

```diff
@@ -677,20 +677,17 @@ let BattleMovedex = {
 		flags: {contact: 1, protect: 1, mirror: 1},
 		beforeTurnCallback: function (pokemon, target) {
 			pokemon.addVolatile('assurance');
-			pokemon.volatiles.assurance.position = target.position;
+			/** @type {boolean?[]} */
```

Comment:
You can't use <code>@type</code> this way. It needs to go in the <code>Pokemon</code> constructor, but <code>volatiles</code> is not currently typed anyway.